src/query/v2/plan/operator.cpp

```diff
@@ -247,6 +258,65 @@ class DistributedCreateNodeCursor : public Cursor {
     return requests;
   }
 
+  void PlaceNodesOnTheMultiFrame(MultiFrame &multi_frame, ExecutionContext &context) {
+    auto multi_frame_reader = multi_frame.GetValidFramesConsumer();
```

Comment:
GetValidFramesModifier instead of GetValidFramesConsumer since you are just modifying the content of the frame and not its validity